### PR TITLE
mpd 0.23.13: Enable Wavpack feature

### PR DESCRIPTION
**Pull request by t0maz**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [x] Have you ensured that your commits follow the [commit style guide](https://docs.brew.sh/Formula-Cookbook#commit)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `HOMEBREW_NO_INSTALL_FROM_API=1 brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `HOMEBREW_NO_INSTALL_FROM_API=1 brew install --build-from-source <formula>`)? If this is a new formula, does it pass `brew audit --new <formula>`?

-----

### CODE REVIEW

File: Formula/m/mpd.rb
Comment (t0maz) [reply Formula/m/mpd.rb:75]: It can be verified by checking the presence of "[wavpack] wv" in the "Decoders plugins:" list of the "mpd --version" command output.

File: Formula/m/mpd.rb
Comment (t0maz) [reply Formula/m/mpd.rb:7]: I'm just a newbie here but the change to be effective needs a rebuild of MPD and since the updated formulea will provide now updated MPD binary shouldn't it have a different revision or any other mark of difference vs the previous formulea?

Comment (ZhongRuoyu) [reply Formula/m/mpd.rb:7]: A revision bump is not necessary for a rebuild. We only bump the revision number when it's necessary, because that would force every `mpd` user to download and install the new update, for which we have received some complaints in the past.

Without a revision bump, it's still possible to install the rebuilt formula with `brew reinstall mpd` after this gets merged.

File: Formula/m/mpd.rb
Comment (SMillerDev) [reply Formula/m/mpd.rb:75]: Can you add `assert_match "[wavpack] wv", shell_output("mpd --version")` to the test block then?

Comment (t0maz) [reply Formula/m/mpd.rb:75]: Please find it added in the latest commit.

File: Formula/m/mpd.rb
Comment (SMillerDev) [inline Formula/m/mpd.rb:110]:
```suggestion
    assert_match "[wavpack] wv", shell_output("#{bin}/mpd --version")
```

**Review by MikeMcQuaid — DISMISSED** [review]

**Comment by MikeMcQuaid** [comment]
@t0maz to avoid the need for the replacement PR: can you allow maintainers/bots to push to your PRs? Thanks!

**Comment by t0maz** [comment]
Many thanks guys for review, your help and approval!

**Comment by MikeMcQuaid** [comment]
@t0maz whoops, ignore my comment above, I misunderstood.

**Pull request closed without merging**
